### ISSUE TITLE
Ferals should take heat damage same as you

### ISSUE THREAD

**Issue by Heitomos** [CleverRaven/Cataclysm-DDA#56349]
### Is your feature request related to a problem? Please describe.

I can't get within 4 spaces of a burning building without taking damage to all limbs, yet Ferals will stand next to an inferno for an hour and be fine.

### Solution you would like.

A way for Hot Air to affect Ferals same as it would you/other humans.

### Describe alternatives you have considered.

Some form of story explanation for why they are immune to heat.

**Comment by Night-Pryanik** [comment]
Essentially the same as #30072.

**Comment by AcidAntOnAMinefield** [comment]
Every creature, really, not just ferals. Zombies or otherwise.

**Comment by Heitomos** [comment]
> Every creature, really, not just ferals. Zombies or otherwise.

Yeah, I agree. I had just posted this after watching Ferals roasting marshmallows for a while.
